src/topology/algebra/group.lean

```diff
@@ -678,3 +678,13 @@ instance additive.topological_add_group {G} [h : topological_space G]
 instance multiplicative.topological_group {G} [h : topological_space G]
   [add_group G] [topological_add_group G] : @topological_group (multiplicative G) h _ :=
 { continuous_inv := @continuous_neg G _ _ _ }
+
+section units
```

Comment:
Let's make this `namespace units` actually (but you'll need to check that there are no other namespaces open, and move it if so.  Same in `topology.algebra.monoid`.

Reply:
Sure even if I guess in group nothing will change as the instance is already automatically called units.topological_space